spec/models/user_spec.rb

```diff
@@ -17,5 +17,16 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'callbacks' do 
+    context 'validation before saving the user' do
+      
+      user = FactoryBot.build(:user_without_uid)
```

Comment:
Lets try to use `let` variables in specs
https://pololu.github.io/rpicsim/file.IntroductionToRSpec.html#label-Let+variables

Also, i think you can do
`let!(:user) { create(:user_without_uid) } `

Reply:
Thanks for the suggestion.